docs/COMPONENT_GUIDELINES.md

```diff
@@ -29,3 +29,80 @@ So solutions like `@media-queries` might not work, and `window` shouldn't be dir
 
 * `wix-ui-tpa` are rendered for Wix's users-of-users, and so they must comply with web accessibility
  principles.
+ 
+ ## Typography
+ When creating a new component that renders text, we should in most cases use the `<Text/>` component with the correct props and style.  
+ In some cases we might need to use a typography that isn't defined in the `<Text/>` component.  
+ In this case, we rely on the style processor's syntax, to ensure we get the correct font from the viewer.
+ 
+ 
+ ### How it works
```

Comment:
I find this section a bit confusing.
As a consumer - I'd expect the simple example of how to extend the Text component and right afterward a link to the style processor repo. Also, consider pointing to https://github.com/wix-incubator/tpa-style-webpack-plugin.

If you want to have a section that talks about how the style-processor works, I wouldn't mix it here in the typography but have a different file

Reply:
This isn't a consumer doc.
I moved the original doc in the PR to here, to the contribution documentation

Reply:
the whole purpose of this task was to clarify the way we write fonts when contributing